src/beast/beast/threads/impl/Stoppable.test.cpp

```diff
@@ -210,10 +215,11 @@ class Stoppable_test
         void onChildrenStopped() override
         {
             stop_ = please_stop;
-            while (stop_ != have_stopped)
+            while (stop_ != stopping)
                 ;
             Stoppable::stopped();
             test_.expect(--test_.count == 1, "A::onChildrenStopped called out of order");
+            stop_ = stopped;
```

Comment:
At first I was puzzled why `A` was the only class exhibiting the stopping problem.  I eventually realized that `A` is the only object running on a different thread from `Root`.  So now I see why the fix is limited to `A`.